llvm/lib/SYCLLowerIR/LowerESIMD.cpp

```diff
@@ -126,8 +126,8 @@ struct ESIMDIntrinDesc {
                                 // -1 denotes return value
         GenXArgConversion Conv; // GenXArgConversion
       } Arg;
-      int NRemArgs;           // SRC_CALL_ALL: number of remaining args
-      unsigned int ArgConst;  // CONST_I16 OR CONST_I32: constant value
+      int NRemArgs;          // SRC_CALL_ALL: number of remaining args
```

Comment:
unrelated change - please remove

Reply:
Removed in https://github.com/intel/llvm/pull/4014/commits/5f8dfa54758be4bc71d9169091a22e45d449a961